### DISCUSSION TITLE
Error: More than undefined iterators used concurrently

### DISCUSSION THREAD

**Discussion by Congenial-GmbH** [requarks/wiki#5029] [Error / Bug Report] [answered]
Hi

When I create too many nested tabsets (cf. example below), I get the following error message:

> Error when running ob render-page: This NodeIterator is no longer working. More than undefined iterators are being used concurrently. You can increase the 'conturrentNoteIterators' option to make This error go away.

Where can I configure the `concurrentNoteIterators` parameter?

```
# Test

# {.tabset}

## 1
### {.tabset}

#### a

x

#### b

x

## 2
### {.tabset}

#### a

x

#### b

x

## 3
### {.tabset}

#### a

x

#### b

x

## 4
### {.tabset}

#### a

x

#### b

x

## 5
### {.tabset}

#### a

x

#### b

x
```

**Comment by NGPixel** [comment, marked as answer]
You can't use tabsets within tabsets...

**Reply by rednairb** [reply]
I just ran into this as well, and agree it would be nice if it was supported.  It sure helps to keep information organized!  it seems to work for a bit as I have 3 nested tabsets, with 3 instances of the 3rd layer.  When I try to add the forth instance at the 3rd layer is when it craps out.  Any reason why you don't want to support this?

**Reply by NGPixel** [reply]
It's not that I don't want to support, it simply wasn't designed to handle nested tabs.

**Comment by Congenial-GmbH** [comment]
Thanks for your feedback.

The example below shows that nested tabs are actually working, which is a bit confusing for me. In my view, it would be grate if nested tabs were officially supported.

```
# Nested Tabsets

# {.tabset}

## Suptab A

Text **above** A's sub-tabs

### {.tabset}

Text **below** A's sub-tabs

#### Subtab A1

Content for **sub-tab A1**

#### Subtab A2

Content for **sub-tab A2**

## Suptab B

Text **above** B's sub-tabs

### {.tabset}

Text **below** B's sub-tabs

#### Subtab B1

Content for **sub-tab B1**

#### Subtab B2

Content for **sub-tab B2**

#### Subtab B3

Content for **sub-tab B3**
```
